app/models/authentication_request_header.rb

```diff
@@ -0,0 +1,14 @@
+class AuthenticationRequestHeader < Authentication
+
+  def ansible_config_format
```

Comment:
This function takes all the fields from the db and translate them into a hash that we place inside of the yaml file used when deploying ansible playbook, makes sense? name suggestions?